engines/quickjs/extract.js

```diff
@@ -30,11 +29,23 @@ const extract = ({ filePath, binary, os }) => {
 			engine: binary,
 			path: tmpPath,
 		});
-		switch (os) {
-			case 'linux64': {
-				installer.installBinary({ 'qjs': binary });
-				break;
-			}
+		if (os.startsWith('win')) {
+			installer.installBinary(
+				{ 'qjs.exe': `${binary}.exe` },
+				{ symlink: false }
+			);
+      installer.installLibrary('libwinpthread-1.dll');
+			installer.installScript({
+				name: `${binary}.cmd`,
+				generateScript: (targetPath) => {
+					return `
+						@echo off
+						"${targetPath}\\${binary}.exe" %*
+					`;
+				}
+			});
+		} else {
```

Comment:
We’d hit this `else` case on macOS as well, even though macOS binaries aren’t available: https://bellard.org/quickjs/binary_releases/

So I think we want to stick to `switch (os)` here, and we could just share the branch between `win32` and `win64`.